plugins/DevicesDetection/Controller.php

```diff
@@ -13,26 +13,52 @@
 use Piwik\Db;
 use Piwik\Piwik;
 use Piwik\Plugin\ControllerAdmin;
+use Piwik\Plugin\Manager AS PluginManager;
+use Piwik\Plugins\DevicePlugins\Reports\GetPlugin;
 use Piwik\Plugins\DevicesDetection\Reports\GetBrand;
 use Piwik\Plugins\DevicesDetection\Reports\GetBrowserEngines;
 use Piwik\Plugins\DevicesDetection\Reports\GetBrowsers;
 use Piwik\Plugins\DevicesDetection\Reports\GetModel;
 use Piwik\Plugins\DevicesDetection\Reports\GetOsVersions;
 use Piwik\Plugins\DevicesDetection\Reports\GetType;
+use Piwik\Plugins\Resolution\Reports\GetConfiguration;
+use Piwik\Plugins\Resolution\Reports\GetResolution;
 use Piwik\View;
 
 class Controller extends \Piwik\Plugin\Controller
 {
-    public function index()
+    public function devices()
```

Comment:
to keep BC in case users are [embedding Widgets](http://piwik.org/docs/embed-piwik-report/) in their apps/website, we should define an alias from index() to devices()